cmd/ocm/account/status/cmd.go

```diff
@@ -87,9 +80,13 @@ func run(cmd *cobra.Command, argv []string) error {
 	// Display user and which server they are logged into
 	currAccount := response.Body()
 	currOrg := currAccount.Organization()
-	fmt.Printf("User %s on %s in org '%s' %s (external_id: %s)",
+	fmt.Printf("User %s on %s in org '%s' %s (external_id: %s) ",
 		currAccount.Username(), cfg.URL, currOrg.Name(), currOrg.ID(), currOrg.ExternalID())
 
+	if urlOverride := os.Getenv("OCM_URL"); urlOverride != "" {
+		fmt.Printf("- OCM_URL overridden via env to %s - ", urlOverride)
+	}
```

Comment:
Changes the output based on whether the env is overridden:

```
> ./ocm account status
User testuser on https://api.openshift.com in org 'Red Hat' aaabbbcccdddeeefffggg (external_id: 123456789) Roles: Role1, Role2, AuthenticatedUser

ocm-cli on  api-url-from-env [!]  staging
> OCM_URL=https://regional.api.openshift.com ./ocm account status
User testuser on https://api.openshift.com in org 'Red Hat' aaabbbcccdddeeefffggg (external_id: 123456789) - OCM_URL overridden via env to https://regional.api.openshift.com - Roles: Role1, Role2, AuthenticatedUser```

Reply:
Happy to handle this differently if there's any better ideas of how to format this.

Reply:
imo just nix it. your debug warning when you override the url in ConnectionBuilder covers all use cases and doesnt create noise.